README.md

```diff
@@ -1 +1,10 @@
 Docker base for node projects
+
+## how to tag a new image
+
+- modify Dockerfile to taste
+- build image from new dockerfile
+- tag it (example below)
+  - `docker tag 0e5574283393 rentpath/rp_node:node-v14.15.4_yarn-v1.22.5`
```

Comment:
I'd suggest building in tagging in one step, but what you have is fine. If you build and tag in one step, here's how:
```
docker build . -t rentpath/rp_node:node-v14.15.4_yarn-v1.22.5
```

Reply:
i like that much better. will do this update and merge